testutils/modulecontext/module_context_utils.go

```diff
@@ -0,0 +1,22 @@
+//go:build !release
+
+package modulecontext
+
+import (
+	"context"
+	"github.com/TBD54566975/ftl/internal/modulecontext"
+)
+
+type SingleContextSupplier struct {
+	moduleCtx modulecontext.ModuleContext
+}
+
+func MakeDynamic(ctx context.Context, m modulecontext.ModuleContext) *modulecontext.DynamicModuleContext {
+	supplier := modulecontext.ModuleContextSupplier(SingleContextSupplier{m})
+	result, _ := modulecontext.NewDynamicContext(ctx, supplier, "test")
```

Comment:
Should this error be ignored? I think if it is, put in a comment. I'll update the linter to check this.

Reply:
it shouldn't happen since the supplier doesn't have a failure path but I've added a panic just in case